mlir/include/mlir/Dialect/ArmSME/IR/ArmSMEOps.td

```diff
@@ -489,22 +492,27 @@ def StoreTileSliceOp : ArmSME_Op<"store_tile_slice"> {
     dimensions since the operation is scalable, and the element type must be a
     scalar that matches the element type of the input tile.
 
+    An SSA value `mask` specifies to mask out elements written to the MemRef.
+    The `mask` type is an `i1` vector with a shape that matches how elements
+    are written to the MemRef.
```

Comment:
nit: I don't think the we need to say "SSA value" (it's already clear from that it's an operand and not an attribute). 

I also think this description could be simplified: 
```suggestion
    The provided `mask` is used to specify which elements of the tile slice will be stored.
```

This makes it clear it's masking the tile slice. The other details like it's writing to a `MemRef`, or that the mask is `i1`s should already be clear from context. :)

Reply:
Yeah that's a lot clearer cheers, I was lazy and copied this from `transfer_write`